open-api/rest-catalog-open-api.yaml

```diff
@@ -3260,6 +3260,28 @@ components:
           additionalProperties:
             type: string
 
+    FineGrainedDataProtectionRules:
+      type: object
+      description: >
+        Fine-grained data protection rules for a table as result of fine grained policy evaluation at the catalog end based on the clients access rights.
+        
+        The client SHOULD use these rules to enforce fine-grained data protection like column and row level access when reading data from the table.
+      properties:
+        projections:
+          description: >
+            This field contains a list of columns or column transforms to be projected.
+            
+            Note: That each column must have only a single projection, meaning a column can be projected as-is or as a transformed value, but not both.
+          type: array
+          items:
+            $ref: '#/components/schemas/Term'
+        row-filter:
+          description: >
+            An expression that filters rows. Only rows for which the expression evaluates to true should be allowed to read.
+            If the catalog supports multiple row access filter against the table, its the catalogs responsibility to combine them with
```

Comment:
Nit: "its" is possessive. Use "it is" instead.

I think that only allowing one row filter also implies this, but it is okay to have it here.